src/components/widget/List/List.js

```diff
@@ -349,7 +349,7 @@ ListWidget.propTypes = {
   mainProperty: PropTypes.object,
   isModal: PropTypes.bool,
   autoFocus: PropTypes.bool,
-  selected: PropTypes.object,
+  selected: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
```

Comment:
I wish we could force this to always be an array. But probably without types and with our current test coverage it'd be a nightmare :)

Reply:
When is solo item selection (from dropdown) is an object.   When the selection comes from the multiselect si an array of objects